src/main/java/de/tum/in/www1/artemis/domain/ExerciseGroup.java

```diff
@@ -0,0 +1,174 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import de.tum.in.www1.artemis.domain.exam.Exam;
+
+@Entity
+@Table(name = "exercise_group")
+public class ExerciseGroup extends AbstractAuditingEntity {
+
+    // region CONSTRUCTORS
+    // -----------------------------------------------------------------------------------------------------------------
+    // no arg constructor required for jpa
+    public ExerciseGroup() {
+    }
+
+    public ExerciseGroup(Long id, String title, Boolean isMandatory, Integer positionInExam, Exam exam, Set<Exercise> exercises) {
+        this.id = id;
+        this.title = title;
+        this.isMandatory = isMandatory;
+        this.positionInExam = positionInExam;
+        this.exam = exam;
+        this.exercises = exercises;
+    }
+
+    // -----------------------------------------------------------------------------------------------------------------
+    // endregion
+
+    // region BASIC PROPERTIES
+    // -----------------------------------------------------------------------------------------------------------------
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "title")
+    private String title;
+
+    /**
+     * Mandatory exercise groups need to be included in the exam
+     */
+    @Column(name = "is_mandatory", nullable = false)
+    private Boolean isMandatory = true;
+
+    /**
+     * Specifies the position of this exercise group if it is used in an exam
+     */
+    @Column(name = "position_in_exam", nullable = false)
+    private Integer positionInExam = 1;
```

Comment:
please remove this attribute. We should rather use an ordered collection for Exam --> ExerciseGroup